### PR TITLE
Add CloudStack getting started docs

### PR DESCRIPTION
**Pull request by chrisnegus**
*Issue #*: [3658](https://github.com/aws/eks-anywhere/issues/3658)

*Description of changes:*
Adding a CloudStack getting started guide to the EKS Anywhere docs.

### CODE REVIEW

File: docs/content/en/docs/getting-started/production-environment/cloudstack-getstarted.md [hunk 1]
```diff
@@ -3,6 +3,226 @@ title: "Create CloudStack production cluster"
...
+
+1. Generate a workload cluster config:
+   ```bash
+   CLUSTER_NAME=w01
+   eksctl anywhere generate clusterconfig $CLUSTER_NAME \
+      --provider cloudstack > eksa-w01-cluster.yaml
+   ```
+
+   Refer to the initial config described earlier for the required and optional settings.
+   Ensure workload cluster object names (`Cluster`, `CloudDatacenterConfig`, `CloudStackMachineConfig`, etc.) are distinct from management cluster object names.
```
Comment (vignesh-goutham) [inline docs/content/en/docs/getting-started/production-environment/cloudstack-getstarted.md:184]: Should we here say, to make sure that the managementCluster field on the cluster spec is updated to the right cluster name?

**Review by maxdrib — APPROVED** [review]
Looks good to me! Excellent notes, thank you for taking time to understand the product and writing these so clearly

File: docs/content/en/docs/getting-started/production-environment/cloudstack-getstarted.md (hunk 1 above)
Comment (chrisnegus) [reply docs/content/en/docs/getting-started/production-environment/cloudstack-getstarted.md:184]: Great point! I'll do that.

Comment (chrisnegus) [reply docs/content/en/docs/getting-started/production-environment/cloudstack-getstarted.md:184]: Done.

**Review by vignesh-goutham — APPROVED** [review]
/approve
/cherrypick release-0.12

**Pull request merged**